.circleci/config.yml

```diff
@@ -12,9 +12,8 @@ jobs:
       # Specify service dependencies here if necessary
       # CircleCI maintains a library of pre-built images
       # documented at https://circleci.com/docs/2.0/circleci-images/
-      # - image: circleci/postgres:9.4
```

Comment:
Would it make sense remove all of these comments too, unless you'll be starting some services?

Reply:
done